### ISSUE TITLE
support yaml fix

### ISSUE THREAD

**Issue by marcelldls** [epics-containers/ibek#243]
https://github.com/epics-containers/ibek-support/blob/8d18d58c19aa4443aaf7e9df7e81eb4e141b9774/ADAravis/ADAravis.ibek.support.yaml#L139-L142

**Comment by marcelldls** [comment]
@gilesknap Is this the feature that currently prevents multiple AutoADGenICam.pvi.device.yaml?

**Comment by gilesknap** [comment]
Yes we just need to jinja templating of this field like we do for database.file

**Comment by coretl** [comment]
I think we need to fix this by generating both db and pvi file from `makeDb.py` (or similar) rather than running the db file through `pvi convert`

**Comment by coretl** [comment]
Not required as we will do https://github.com/epics-containers/ioc-adaravis/issues/42
